storage/factory/pruningStorerFactory.go

```diff
@@ -566,14 +577,15 @@ func (psf *StorageServiceFactory) createPruningPersister(arg *pruning.StorerArgs
 }
 
 func (psf *StorageServiceFactory) initOldDatabasesCleaningIfNeeded(store dataRetriever.StorageService) error {
-	shouldCleanOldData := psf.generalConfig.StoragePruning.CleanOldEpochsData && !psf.prefsConfig.FullArchive
-	if !shouldCleanOldData {
+	isFullArchive := !psf.prefsConfig.FullArchive
```

Comment:
why isFullArchive is set false for full archive?

Reply:
good catch :medal_sports:  fixed the naming